parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -106,9 +106,15 @@ mobyPkgVersion() {
 }
 installRunc() {
   local v
+  local url
   v=$(runc --version | head -n 1 | cut -d" " -f3)
-  if [[ $v != "1.0.3" ]]; then
-    apt_get_install 20 30 120 moby-runc=1.0.3* --allow-downgrades || exit 27
+  if [[ $v != "1.1.2" ]]; then
+    url=${MS_APT_REPO}/ubuntu/${UBUNTU_RELEASE}/multiarch/prod/pool/main/m/moby-runc/moby-runc_1.1.2%2Bazure-ubuntu${UBUNTU_RELEASE}u1_amd64.deb
```

Comment:
E: Version '1.1.2*' for 'moby-runc' was not found

Looks like this version doesn't exist yet.

@cpuguy83 in case he knows anything about when this version will be made available

Reply:
Yeah, it's only in the "testing" channel right now.
Considering promoting it to stable since the 1.0 tree is no longer active.

I don't currently have a process for promoting a release from "testing" to "stable". I need to work this out.

Reply:
Thx @cpuguy83 and @jackfrancis